File: codegen/codegen-erlang/src/test/resources/ir/to_binary_rest_json.expected.erl
to_binary(V) when is_binary(V) -> V;
to_binary(V) when is_list(V) -> list_to_binary(V);
to_binary(true) -> <<"true">>;
to_binary(false) -> <<"false">>;
to_binary(V) when is_atom(V) -> atom_to_binary(V, utf8);
to_binary(V) when is_integer(V) -> integer_to_binary(V);
to_binary(V) when is_float(V) -> float_to_binary(V).

File: codegen/codegen-elixir/src/test/resources/ir/to_binary_rest_json.expected.ex
defp to_binary(v) when is_binary(v), do: v
defp to_binary(v) when is_list(v), do: IO.iodata_to_binary(v)
defp to_binary(true), do: "true"
defp to_binary(false), do: "false"
defp to_binary(v) when is_atom(v), do: Atom.to_string(v)
defp to_binary(v) when is_integer(v), do: Integer.to_string(v)
defp to_binary(v) when is_float(v), do: Float.to_string(v)